pkgs/shells/marcel/changes.patch

```diff
@@ -0,0 +1,83 @@
+diff --git a/experiments/cp_behavior.py b/experiments/cp_behavior.py
+index d2b19ea..dae8898 100644
+--- a/experiments/cp_behavior.py
++++ b/experiments/cp_behavior.py
+@@ -31,7 +31,7 @@ def run(command, expected_err=None):
+     else:
+         process = subprocess.run(command,
+                                  shell=True,
+-                                 executable='/bin/bash',
++                                 executable='/run/current-system/sw/bin/bash',
```

Comment:
this requires bash to be on path, this seems incorrect. if anything, `substituteInPlace` fits better than using a patch.

Reply:
now using substituteInPlace and ${bash}/bin/bash, will you check for me whether i do it right or not?